spec/latest/core/index.html

```diff
@@ -1193,6 +1193,35 @@ <h3>Templated Links</h3>
       </dl>
     </aside>
 
+    <p>The example <i>Description of an IRI Template</i> that was already mentioned
+      uses a <i>variableRepresentation</i> on the whole <i>IriTemplate</i>, but
+      each mapped variable represented as an <i>IriTemplateMapping</i> can have
+      its own <i>variableRepresentation</i> declaration which MUST override
+      the declaration from the <i>IriTemplate</i> level for that very
+      <i>IriTemplateMapping</i> as it takes precedence. The example below
+      depicts such a situation:
+    </p>
+
+    <pre class="example nohighlight" data-transform="updateExample"
+         title="Variable level representation">
+      <!--
+      {
+        "@context": "http://www.w3.org/ns/hydra/context.jsonld",
+        ****"@type": "IriTemplate"****,
+        ****"template": "http://api.example.com/issues{?q}"****,
+        ****"mapping"****: [
+          {
+            ****"@type": "IriTemplateMapping"****,
+            ****"variable": "q"****,
+            ****"property": "hydra:freetextQuery"****,
+            ****"variableRepresentation": "BasicRepresentation"****,
+            ****"required": true****
+          }
```

Comment:
A second variable for good measure

```suggestion
          },
          {
             ****"@type": "IriTemplateMapping"****,
             ****"variable": "category"****,
             ****"property": "schema:category"****
          }
```